### ISSUE TITLE
All the curling quotes seem not to be long/tall enough overall

### ISSUE THREAD

**Issue by EbenSorkin** [googlefonts/MetrophobicFont#8]

**Comment by aoifemooney** [comment]
yep, you definitely are. They definitely were too short, but I had fixed them in the latest version. The top is the latest version, the bottom is the previous version.
<img width="836" alt="screen shot 2018-02-25 at 16 36 06" src="https://user-images.githubusercontent.com/12103183/36646752-05dc5278-1a4a-11e8-8fde-bb7e4b182e8a.png">
<img width="802" alt="screen shot 2018-02-25 at 16 35 55" src="https://user-images.githubusercontent.com/12103183/36646753-05eaf76a-1a4a-11e8-9564-bb6f16cf88c6.png">
